Tallies/TallyResponses/macroResponse_class.f90

```diff
@@ -119,12 +119,16 @@ function get(self, p, xsData) result(val)
     real(defReal)                         :: val
     type(neutronMacroXSs)                 :: xss
     class(neutronMaterial), pointer       :: mat
+    character(100), parameter :: Here = 'get (macroResponse_class.f90)'        
 
     val = ZERO
 
     ! Return zero if particle is not neutron or if the particle is in void
     if (p % type /= P_NEUTRON) return
     if (p % matIdx() == VOID_MAT) return
+    if ((self % MT == macroEnergyDepoZero) .and. (p % isMG .eqv. .true.)) then
```

Comment:
Perhaps for another PR (maybe an issue for now), but it is conceivable to have an MG equivalent. Basically this would mean giving the MG data one more input and it would be straightforward to compute qFission in a given material.